rust/generate_message/tests/make.rs

```diff
@@ -0,0 +1,35 @@
+pub mod common;
+use crate::common::{assert_cmd_stdout, assert_files_eq, remove_if_exists, run_cmd_test};
+
+use constants::{EXPORT_FOLDER, FOLDER};
+use std::fs;
+
+#[test]
+fn it_signs() {
+    let unsigned = format!("{}/sign_me_load_metadata_polkadotV30", FOLDER);
+    let unverified = format!("{}/load_metadata_polkadotV30_unverified.txt", EXPORT_FOLDER);
+    let signed = format!(
+        "{}/load_metadata_polkadotV30_Alice-sr25519.txt",
+        EXPORT_FOLDER
+    );
+    remove_if_exists(&unverified);
+    remove_if_exists(&unsigned);
+    remove_if_exists(&signed);
```

Comment:
Are all the tests racing on the same directories and file paths? If so maybe they should run in their isolated tmpdirs each?

Reply:
That makes sense. Thanks! Fixed